src/lib/SlaveApiClient.js

```diff
@@ -17,31 +17,57 @@
 
 "use strict";
 
-let xmlrpc = require('xmlrpc');
+let xmlrpc = require('@sixriver/xmlrpc');
 
 //-----------------------------------------------------------------------
 
-class SlaveApiClient {
+class AbortedError extends Error {
+    constructor() {
+        super();
+
+        Object.setPrototypeOf(this, AbortedError.prototype);
```

Comment:
Maybe add comment here about why this is needed (instanceof plus older version of js)?

Reply:
yea :(